### PR TITLE
Add system/unique_board_id

### PR DESCRIPTION
**Pull request by Wren6991**
Example of accessing the unique 64 bit ID of external flash, which can be used as a unique board identifier

### CODE REVIEW

**Comment by Wren6991** [comment]
See #6

File: system/unique_board_id/unique_board_id.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
```
Comment (lurch) [inline system/unique_board_id/unique_board_id.c:2]: 2021? :wink:

Comment (Wren6991) [reply system/unique_board_id/unique_board_id.c]: Fixed via force push, thanks

**Comment by Wren6991** [comment]
Note this depends on https://github.com/raspberrypi/pico-sdk/pull/12

**Comment by Wren6991** [comment]
Now updated to use the safe `pico/unique_id.h` accessor

**Pull request closed without merging**